client-websocket/src/main/java/com/github/twitch4j/client/websocket/WebsocketConnection.java

```diff
@@ -172,6 +182,9 @@ protected void setState(WebsocketConnectionState newState) {
     public void connect() {
         WebsocketConnectionState connectionState = this.connectionState.get();
         if (connectionState == WebsocketConnectionState.DISCONNECTED || connectionState == WebsocketConnectionState.RECONNECTING || connectionState == WebsocketConnectionState.LOST) {
+            if (closed.get())
+                throw new IllegalStateException("WebsocketConnection was already closed!");
```

Comment:
improbable breaking change: `connect()` can no longer be called after `close()`

Reply:
the instance shouldn't be usable after calling `close()`, so adding this is good